picard/ui/mainwindow.py

```diff
@@ -182,6 +184,8 @@ class MainWindow(QtWidgets.QMainWindow, PreserveGeometry):
         TextOption("persist", "current_directory", ""),
         FloatOption("persist", "mediaplayer_playback_rate", 1.0),
         IntOption("persist", "mediaplayer_volume", 50),
+        Option("setting", "file_renaming_scripts", {}),
+        TextOption("setting", "selected_file_naming_script_id", ""),
```

Comment:
These options don't need to be redefined here. Actually they should be only defined once. Currently they are also duplicated in options/renaming.py and in scripteditor.py.

The options usually are defined at the place they get set, so usually in the options. I think here maybe the scripteditor.py is the most fitting place. The only requirement is that this file gets imported on start, so that the option is registered.

Reply:
The existing duplicates where cleaned up by zas in https://github.com/metabrainz/picard/pull/1847

So just removing the above two lines from this PR should be enough.

Reply:
I rebased on the updated `master` branch just in case.